server/api/jurisdictions.py

```diff
@@ -535,6 +536,45 @@ def status(jurisdiction_id: str) -> JurisdictionStatus:
     }
 
 
+@api.route(
+    "/election/<election_id>/jurisdictions/last-login",
+    methods=["GET"],
+)
+@restrict_access([UserType.AUDIT_ADMIN])
+def get_last_login_by_jurisdiction(election: Election):
+    # Look for events after most recent round started, or election creation time
+    # if no rounds exist yet
+    current_round = get_current_round(election)
+    query_timestamp_after = (
+        current_round.created_at if current_round else election.created_at
```

Comment:
Nice! This covers one of the main things that I wanted to check, that we reset this for every phase of the audit